app/src/main/java/me/ccrama/redditslide/util/LinkUtil.java

```diff
@@ -282,19 +243,37 @@ public static void crosspost(Submission submission, Activity mContext) {
         mContext.startActivity(new Intent(mContext, Crosspost.class));
     }
 
-    public static String getPackage(Intent intent) {
+    public static void overridePackage(Intent intent) {
```

Comment:
The flow on this is now

1. Get package name for the given intent
2. Get package name for a browser
3. If the intent package name is equal to Slide's package name, set the packageToSet on the intent to the browser package name
4. If the browser package name is being used, and the user has the browser set up to be overridden, override that package with the user selected one if it still exists on the device
5. If the package is to be overridden, override it.